docs/source/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+
+Sitemap: https://conda.github.io/menuinst/sitemap.xml
```

Comment:
What does https://conda.github.io point to ? I get a 404

Reply:
There's no `conda.github.io` repo, so nothing, but other projects like [constructor](https://conda.github.io/constructor/) or [conda-libmamba-solver](https://conda.github.io/conda-libmamba-solver/) are on that GH Pages domain.